app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -209,8 +209,9 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService)(implic
       case Some(document) =>
         val applicationId = document.getAs[String]("applicationId").get
         val applicationStatus = document.getAs[String]("applicationStatus").get
+        val fastPassReceived = document.getAs[FastPassDetails]("fastpass-details").flatMap(_.fastPassReceived)
         findProgress(applicationId).map { (p: ProgressResponse) =>
-          ApplicationResponse(applicationId, applicationStatus, userId, p)
+          ApplicationResponse(applicationId, applicationStatus, userId, p, fastPassReceived)
```

Comment:
As discussed at the moment its not relevant to progress. We may get an idea when we do the stories related to civil service.